### PR TITLE
v1.3.5 release prep

### PR DESCRIPTION
**Pull request by rajumsys**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 - [Compare to latest release][unreleased]
 
+## [1.3.5] - 2017-03-07
```
Comment (richleland) [inline CHANGELOG.md]: there are links at the bottom of the document that need to be updated as well. See [this commit](https://github.com/SparkPost/python-sparkpost/commit/8808080ed4cfe17eadb0114eb5b7c12fb311e7a7).

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+### Added
+- [#141](https://github.com/SparkPost/python-sparkpost/pull/141) Added validation recipients data type
```
Comment (richleland) [inline CHANGELOG.md]: "Added validation for `recipients` parameter for transmissions."

File: CHANGELOG.md (hunk 1 above)
Comment (rajumsys) [inline CHANGELOG.md]: adding this future link :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -124,6 +129,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Docs on readthedocs.org
 
 [unreleased]: https://github.com/sparkpost/python-sparkpost/compare/v1.3.4...HEAD
```
Comment (richleland) [inline CHANGELOG.md]: this line needs to be updated to 1.3.5

**Pull request merged**